tests/BUILD.bazel

```
@@ -9,3 +9,18 @@ py_test(
         "@pip//:pytest_pkg",
     ],
 )
+
+
+py_test(
+    name = "test_read_input_file",
+    timeout = "short",
+    srcs = ["test_read_input_file.py"],
+    deps = [
+        "//src:read_input_file",
+        "@pip//:pytest_pkg",
+    ],
+    data = [
+        "//tests/resources:faulty_input.in",
+        "//tests/resources:correct_input.in",
```

You could also just do 
```suggestion
        "resources/faulty_input.in",
        "resources/correct_input.in",
```
if you remove the `tests/resources/BUILD.bazel`. I don't see the need to have the resources dir be its own Bazel package atm